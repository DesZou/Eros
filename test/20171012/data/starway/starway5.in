10000 30000 100
9720 23977
468 1815
6267 10372
3383 26287
6968 24189
668 29871
6987 134
4102 9683
721 11596
5930 14263
1524 22300
9461 16503
8128 16267
1604 17523
7285 9652
1695 23125
1135 2125
7356 20444
4100 29716
86 19871
3756 3730
6986 1387
7251 12952
4986 24616
3911 22482
3023 25508
6543 28430
2945 30
6895 26074
773 6142
1484 28470
4274 1788
9563 6424
5293 3939
7086 13375
7019 26765
4361 18671
9402 22808
6201 10998
5137 8882
6912 14633
7118 29067
3258 26215
1069 3718
2359 29964
5392 4432
4128 2142
2133 10618
9496 20553
4963 19999
6942 29445
5479 28365
7143 29579
6092 19131
8424 16774
2531 21809
6986 28319
8416 16253
6056 13956
9160 11438
5408 22844
6185 18386
7093 1470
6398 25705
1506 834
25 24550
4876 2007
7581 5793
2003 15384
1925 21190
2513 28117
8221 16308
372 21360
6630 23405
8602 23865
1830 3683
423 7270
2747 26751
8346 193
8657 4618
5605 7962
3574 2932
8054 4666
4177 4686
2734 22406
4358 13181
5635 5896
8614 22145
95 8529
7307 11880
633 8715
5503 5733
2811 24146
1905 16129
1273 24565
8108 29483
5376 8631
3952 21463
519 10074
8318 10838
