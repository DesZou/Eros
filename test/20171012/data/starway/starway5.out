1379.462304668
